SDKS-3186: Moving iOS and Android samples and re-structuring the repo (#12)
ForgeRock/sdk-sample-apps: README.md

## Samples

Ping provides these samples to help demonstrate SDK functionality/implementation. They are provided "as is" and are not official products of Ping and are not officially supported.

### Integrate with PingOne Advanced Identity Cloud / PingAM:

> To try out the Ping JavaScript SDK please look at one of our samples under [javascript](./javascript/)

Reviewer: I'm not sure about the naming in these three sentences, it's possibly also a bit wordy/repetitive.
Perhaps something like:
`We provide samples of integrating with Advanced Identity Cloud and PingAM for the following platforms:
 - Android
 - iOS
 - JavaScript`

To try out the Ping iOS SDK please look at one of our samples under [iOS](./iOS/)

To try out the Ping Android SDK please look at one of our samples under [android](./android/)

### Integrate with PingOne DaVinci:

- Samples coming soon

## Documentation

> Documentation for the SDKs is provided on [ForgeRock Backstage](https://backstage.forgerock.com/docs/sdks/latest/index.html), and includes topics such as:

Reviewer: ```suggestion
Detailed [documentation](https://docs.pingidentity.com/sdks/latest/sdks/index.html) is provided, and includes topics such as:
```

- Introducing the SDK Features
- Preparing your server for use with the SDKs
- API Reference documentation

## Requirements

> Javascript

Reviewer: I tend to use "JavaScript" everywhere.

Author: Fixed
